internal/request/graphql/schema/types/base.go

```diff
@@ -40,6 +40,28 @@ func BooleanOperatorBlock() *gql.InputObject {
 	})
 }
 
+// BooleanListOperatorBlock filter block for [Boolean] types.
+func BooleanListOperatorBlock(op *gql.InputObject) *gql.InputObject {
+	return gql.NewInputObject(gql.InputObjectConfig{
+		Name:        "BooleanListOperatorBlock",
+		Description: "These are the set of filter operators available for use when filtering on [Boolean] values.",
+		Fields: gql.InputObjectConfigFieldMap{
+			"_any": &gql.InputObjectFieldConfig{
```

Comment:
suggestion: Independent of Islam's suggestion, it might be nice to extract the names to consts in the client/request package and then also reference them in the connor package as well as here.

Reply:
This is out of scope for this PR imo, but I will create an issue for this.